news/4908.bugfix

```diff
@@ -0,0 +1,5 @@
+FIXED 
```

Comment:
Could you change the News File contents to something like the following?

```
Fixed a typo mistake in subversion support
```

As it's currently written, it wouldn't be too helpful for end users of pip (see `NEWS.rst` for where these contents end up). :)

Reply:
Sure